### ISSUE TITLE
Loss Function Wrapper Already Implemented in base TensorFlow

### ISSUE THREAD

**Issue by FirefoxMetzger** [tensorflow/addons#1866]
Going through the code in this repo, I noticed that the `LossFunctionWrapper` is implemented here, and in base tensorflow.

The version here:
https://github.com/tensorflow/addons/blob/f3d0416fbe3f4c607f4c1ebc2b7554a941ff02a2/tensorflow_addons/utils/keras_utils.py#L24

The version in base TensorFlow (v2.2.0):
https://github.com/tensorflow/tensorflow/blob/77245d07d13522a5cb5d060390fffa1894df5bbf/tensorflow/python/keras/losses.py#L210

Is there a particular reason for this? My suggestion would be to depreciate the version here (it can be kept as a wrapper around the TF class for backward compatibility. Any thoughts?

**Comment by WindQAQ** [comment]
Hi, it is not a public API and we aim to remove all private modules from TensorFlow, so we fork a same thing in addons. Feel free to reopen the issue if I missed anything. Thank you!

https://github.com/tensorflow/addons/blob/master/tools/testing/source_code_test.py#L86
